doc/030_preparing_a_new_repo.rst

```diff
@@ -22,8 +22,10 @@ other options. You can skip to the next chapter once you've read the relevant
 section here.
 
 For automated backups, restic accepts the repository location in the
-environment variable ``RESTIC_REPOSITORY``. For the password, several options
-exist:
+environment variable ``RESTIC_REPOSITORY``. restic can also read the repository
```

Comment:
In the rest of the documentation "Restic" is used when starting a sentence (instead of "restic").